contributors.md

```diff
@@ -2,5 +2,4 @@ Contributors
 ============
 
 bananas24 - teacher
-
-Youssef - YuUs01
\ No newline at end of file
+Youssef - YuUs01
```

Comment:
Good job, now I would like to have all dashes in the same column.
For example:
``
bananas24 - teacher
``
``
Youssef   - YuUs01
``